src/components/RoleLink/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useAppSelector } from "@store";
+import { ADMIN, MANAGER, useTypedTranslation, DropDown } from "@shared";
+import { Link } from "react-router-dom";
+
+export default function RoleLink (): JSX.Element {
+  const t = useTypedTranslation();
+  const role = useAppSelector(state => state.user.role);
+  switch (role) {
+    case MANAGER: {
+      return <Link to={"/catalog-for-manager"} className="secondary hover:text-accent-turkus">{t("catalog")}</Link>;
+    }
+    case ADMIN: {
+      return <Link to={"/catalog-for-admin"} className="secondary hover:text-accent-turkus">{t("catalog")}</Link>;
+    }
+    default: {
+      return (
+        <div className='flex justify-between w-[300px]'>
```

Comment:
Please use default utillities from tailwind, for example in this case we have close value: w-80 /* 320px */